vyper/builtin_functions/functions.py

```diff
@@ -981,8 +1043,7 @@ class AsWeiValue(_BuiltinFunction):
         ("kether", "grand"): 10 ** 21,
     }
 
-    def evaluate(self, node):
-        validate_call_args(node, 2)
+    def _check_denomination(self, node):
```

Comment:
rename to `get_denomination`, since it returns the denomination (and is not just called for its side effects)